checker/src/test/java/tests/NullnessLiteComRegTest.java

```diff
@@ -0,0 +1,31 @@
+package tests;
+
+import java.io.File;
+import java.util.List;
+import org.checkerframework.framework.test.CheckerFrameworkPerDirectoryTest;
+import org.junit.runners.Parameterized.Parameters;
+
+/**
+ * JUnit tests for the Nullness Checker -- testing {@code -ANullnessLite} command-line argument that
+ * will not cover other user commands.
+ */
+public class NullnessLiteComRegTest extends CheckerFrameworkPerDirectoryTest {
+
+    public NullnessLiteComRegTest(List<File> testFiles) {
+
+        super(
+                testFiles,
+                org.checkerframework.checker.nullness.NullnessChecker.class,
+                "nullness",
+                "-Anomsgtext",
+                "-AstubWarnIfNotFound",
+                "-ANullnessLite",
+                "-AsuppressWarnings=method.invocation.invalid",
```

Comment:
Why is `"-AsuppressWarnings=method.invocation.invalid"` needed?

Reply:
It is to reveal the bug.

Before fixing, -ANullnessLite will shadow any user passed -AsuppressWarnings. And the test added faild because it reports an unexpected error (method.invocation.invalid), which should be suppressed from user's pointview.